.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
```

Comment:
Cool, I'll remove this.  Looks like `prettier#master` already has cosmiconfig, but the maintainer has been on vacation, so it's still in the process of getting released: https://github.com/prettier/prettier/issues/2589.

Opened up an issue here to get this taken care of: https://github.com/cjoudrey/graphql-schema-linter/issues/32